pkg/webhooks/validation.go

```diff
@@ -70,6 +72,7 @@ func HandleValidation(
 		OldResource:   oldR,
 		Context:       ctx,
 		AdmissionInfo: userRequestInfo,
+		ExcludeGroupRole : dynamicConfig.GetExcludeGroupRole(),
```

Comment:
Also need to pass to mutation policyContext: https://github.com/nirmata/kyverno/blob/c4da7b2a4a222b1787d7dc82727b067d387f4438/pkg/webhooks/mutation.go#L41-L45
Can you test both mutate and validate policy?